internal/host/host_test.go

```diff
@@ -936,7 +958,10 @@ var _ = Describe("UpdateInventory", func() {
 	BeforeEach(func() {
 		db = common.PrepareTestDB(dbName, &events.Event{})
 		dummy := &leader.DummyElector{}
-		hapi = NewManager(getTestLog(), db, nil, nil, nil, createValidatorCfg(), nil, defaultConfig, dummy)
```

Comment:
The reason this change was required is because the manager now actually calls the hardware validator, so nil is insufficient - a real mock is required.